wcomponents-core/pom.xml

```diff
@@ -83,10 +83,12 @@
 			<version>1.1.1</version>
 		</dependency>
 
+
 		<dependency>
-			<groupId>commons-lang</groupId>
-			<artifactId>commons-lang</artifactId>
-			<version>2.3</version>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<version>3.4</version>
```

Comment:
The biggest change from version 2.3 to 3.4 is the package name is ".lang3" so it can exist with older versions.

I noticed the "handler" bars library we are using is already bringing in version 3.1, and the "commons config" library is still bringing in the old 2.3 version.

If we are upgrading commons-lang. It might pay to look at all our libraries, or at least the apache commons.